frontend/src/components/task/task-table.js

```diff
@@ -185,39 +199,55 @@ class CustomPaginationActionsTable extends React.Component {
 
   render () {
     const { classes, tasks } = this.props
-    const { rowsPerPage, page } = this.state
-    const emptyRows = tasks.data.length ? rowsPerPage - Math.min(rowsPerPage, tasks.data.length - page * rowsPerPage) : 0
+    const { data, rowsPerPage, page, sortBy, sortDirection } = this.state
+    const emptyRows = data.length ? rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage) : 0
 
+    const humanise = (text = '') => text[0].toUpperCase() + text.slice(1)
+    const TableColumnTitle = ({
+      field = String(),
+      getter = (record) => Object(record)[field],
+      active = sortBy === field,
+      id = `task.table.head.${field}`,
+      defaultMessage = humanise(field),
+    }) => 
+      <TableSortLabel
+        active={ active }
+        direction={ active ? (sortDirection == 'asc' ? 'desc' : 'asc') : 'desc' }
```

Comment:
ToDo: figure out why `TableSortLabel` shows ⬆️ icon when sorting direction is asc and ⬇️ for desc.
Quick fixed by replacing asc to desc and vice versa.